build/lib/stylelint/vscode-known-variables.json

```diff
@@ -957,6 +957,7 @@
 		"--vscode-action-item-auto-timeout",
 		"--monaco-editor-warning-decoration",
 		"--animation-opacity",
-		"--chat-setup-dialog-glow-progress"
+		"--chat-setup-dialog-glow-progress",
+		"--colors-Effects-Shadow"
```

Comment:
This name seems a bit inconsistent

Reply:
yeah, tbh i think we have an existing one, i just plug and played something from figma